From 724e84d86901a9305948383fe82d6c751377c109 Mon Sep 17 00:00:00 2001
From: Aurelio Colosimo <aurelio@aureliocolosimo.it>
Date: Thu, 25 Sep 2014 16:14:13 +0200
Subject: [PATCH 3/3] yun_linino_defconfig: temporarly disable
 CONFIG_GPIO_PCA953X_IRQ

CONFIG_GPIO_PCA953X_IRQ is not yet fully stable on mcuio system, so we
do not enable it in yun_linino_defconfig.

Signed-off-by: Aurelio Colosimo <aurelio@aureliocolosimo.it>
---
 arch/mips/configs/yun_linino_defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/configs/yun_linino_defconfig b/arch/mips/configs/yun_linino_defconfig
index b5e545a..b48301b 100644
--- a/arch/mips/configs/yun_linino_defconfig
+++ b/arch/mips/configs/yun_linino_defconfig
@@ -1850,7 +1850,7 @@ CONFIG_GPIO_MCUIO=y
 # CONFIG_GPIO_MAX7300 is not set
 # CONFIG_GPIO_MAX732X is not set
 CONFIG_GPIO_PCA953X=y
-CONFIG_GPIO_PCA953X_IRQ=y
+# CONFIG_GPIO_PCA953X_IRQ is not set
 CONFIG_GPIO_PCF857X=y
 # CONFIG_GPIO_SX150X is not set
 # CONFIG_GPIO_ADP5588 is not set
-- 
1.7.9.5

